### PR TITLE
CTP-2245 code review

### PR DESCRIPTION
**Pull request by aydevworks**
Related to Jira story: https://ucldata.atlassian.net/browse/CTP-2245
Overall the code was good to me. I would suggest some minor changes as I commented below.
May be add a README.md to describe what this plugin does?

### CODE REVIEW

File: block_news.php [hunk 1]
```diff
@@ -29,19 +29,17 @@ class block_news extends block_base {
      * @return void
      */
     public function init() {
-        global $USER;
```
Comment (aydevworks) [inline block_news.php:32]: It's not used, so could be removed.

File: block_news.php [hunk 2]
```diff
@@ -29,19 +29,17 @@ class block_news extends block_base {
...
     /**
      * Gets the block settings.
      *
+     * @return void
      */
     public function specialization() {
-        if (isset($this->config->title)) {
-            $this->title = $this->title = format_string($this->config->title, true, ['context' => $this->context]);
-        } else {
-            $this->title = get_string('pluginname', 'block_news');
```
Comment (aydevworks) [inline block_news.php:44]: `isset($this->config->title)` is set when the value is an empty string. e.g. when user add the title and then later removed it.
I think it's better to check for empty string using `empty()`.

Removed the extra `$this->title`.

`else {
            $this->title = get_string('pluginname', 'block_alerts');
    }`
this is not necessary as it is set in the `init()` already.

File: templates/content.mustache [hunk 3]
```diff
@@ -83,7 +83,7 @@
 
 <!-- We use an inline style tag because moodle cannot process @ symbol in css files. -->
 <style>
-/* Swap display between carosel and grid. */
```
Comment (aydevworks) [inline templates/content.mustache:86]: spelling (carousel)

File: block_news.php [hunk 4]
```diff
@@ -162,4 +160,3 @@ public function has_config() : bool {
         return true;
     }
 }
-
```
Comment (aydevworks) [inline block_news.php:165]: Remove additional blank line, same for other files below.

**Review by stuartlamour — APPROVED** [review]
All good, thanks @aydevworks

**Pull request merged**